packages/generator-electrode/generators/app/index.js

```diff
@@ -169,6 +170,13 @@ module.exports = generators.Base.extend({
         },
         {
           type: "confirm",
+          name: "autoSsr",
```

Comment:
the CI should test this flag of the generator. https://github.com/electrode-io/electrode/blob/master/gulpfile.js#L61-L74